### PR TITLE
Simplify mountain connector mixin

### PR DESCRIPTION
**Pull request by wjramos**

### CODE REVIEW

File: src/less/mixins/images.less [hunk 1]
```diff
@@ -216,52 +213,49 @@
...
+
+    &:before,
+    &:after {
+        content:  "";
         position: absolute;
-        border-left: @first-mountain-size solid transparent;
-        border-right: @first-mountain-size solid transparent;
-        border-bottom: @first-mountain-size solid;
+
+        border-top-width:    0 !important;
```
Comment (mhewson) [inline src/less/mixins/images.less:249]: wha whaa whaaa

Comment (wjramos) [inline src/less/mixins/images.less:249]: The element will never have a top border, so we can consolidate common styles to other sides without affecting overall height

**Comment by wjramos** [comment]
Example

**Comment by mhewson** [comment]
approve

[Approved with PullApprove](https://pullapprove.com/rei/rei-cedar/pull-request/88/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

**Pull request merged**